packages/orchestrator-ui-components/src/utils/getEnvironmentVariables.ts

```diff
@@ -0,0 +1,33 @@
+import process from 'process';
+
+// By convention, only this function should be used to access the process.env object.
+// It trows an error if any of variables is not set
```

Comment:
```suggestion
// It throws an error if any of variables is not set
```

Reply:
This reminds me that that comment is a bit outdated. Thanks!